ec_ms_quantification/EC-MS_ixdat_tutorial.ipynb

```diff
@@ -0,0 +1,1017 @@
+    "    tspan_bg=(0, 100),\n",
+    "    ax=\"new\",\n",
+    "    axes_measurement=axes_a, # to highlight the integrated areas on the plot defined above\n",
+    "    return_ax = True # if True, returns the calibration curve axis as a second element\n",
+    ") \n",
+    "# NOTE, it uses and highlights the electrochemical current, not the calculated current density - FIXME!!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "53d6f687",
+   "metadata": {},
+   "source": [
+    "Because we asked for ``return_ax``, two arguments are returned:\n",
+    "The first, which we call ``cal_result_H2``, is a ``MSCalResult`` for H2.\n",
+    "The second, which we call ``ax_H2``, is the axis where it plots the calibration curve.\n",
+    "As we asked for ``axes_measurement``, to be plotted on ``axes_a``, the areas that were integrated are highlighted on that axes.\n",
+    "\n",
+    "The attribute ``cal_result_H2.F`` is the slope of the calibration curve, which is the sensitivity factor in C/mol.\n",
+    "(Note that this calibration_curve works with integrals rather than simply the rates. The latter is not yet implemented in ixdat.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f1113630",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(cal_result_H2)  # prints: MSCalResult(name=\"H2@M2\", mol=\"H2\", mass=\"M2\", F=0.34)\n",
+    "\n",
+    "# This is the first of several MSCalResults, which we will collect in this list:\n",
+    "cal_results = [cal_result_H2]\n",
+    "\n",
+    "# save the figure:\n",
+    "fig_b = ax_b.get_figure()\n",
+    "fig_b.savefig(\"hydrogen_calibration.png\")  # you can use eg .svg instead for vector graphics."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7eb9e86c",
+   "metadata": {},
+   "source": [
+    "Tip: instead of selecting the time spans manually, you can also use the option of passing a ``selector_name (str)`` (Name of selector which identifies the periods of steady electrolysis for automatic selection of timespans of steady electrolysis. E.g. ``\"selector\"`` or ``\"Ns\"`` for biologic EC data) and a ``selector_list`` instead of the ``tspan_list``. Note that you need to have information on the different parts of the EC data in your measurement data to take advanatge of this. If you used a Zilien version < 2.6.0 when running the measurement, this requires that you import the EC data from the ECLab. Here, we can use the data imported to ``meas_combi``.\n",
+    "\n",
+    "First, let's figure out what we should pass to selector list. To this end, we can plot the selector we want to use in our regular EC-MS plot, for example instead of the current: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e3635d7a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "axes_a2 = meas_combi.plot_measurement(tspan=(0, 4700), J_name=\"selector\")"
```

Comment:
I did that, but afterwards realised it was probably not necessary. it shouyld have worked just as well with the original `meas` as imported from zilien. the only reason to use the one with biologic EC data would be because the data is less noisy, but that shouldnt make much of a difference here. Anyway, it's done.